rh-dev.yml

```diff
@@ -0,0 +1,35 @@
+version: "3.4"
+services:
+  pubsub-emulator:
+    container_name: pubsub-emulator
+    image: europe-west2-docker.pkg.dev/ssdc-rm-ci/docker/gcloud-pubsub-emulator:latest
+    ports:
+      - "8538:8538"
```

Comment:
I don't think we should duplicate the pubsub-emulator container config, better to split out the common dependencies on their own.